cubic-server/World.cpp

```diff
@@ -251,3 +251,29 @@ thread_pool::Pool &World::getGenerationPool() { return _generationPool; }
 Seed World::getSeed() const { return _seed; }
 
 uint8_t World::getRenderDistance() const { return _renderDistance; }
+
+long World::getTime() const { return _time; }
+
+long World::getAge() const { return _age; }
+
+/*
+**  Used in the /time command.
+**  Adds time to the current world time
+*/
```

Comment:
The doc comments should be in the hpp.

Reply:
What ? But it is the comment refering to the following function, shoudn't it be here ?